com.unity.cinemachine/Samples~/3D Samples/Clearshot/LeadShot.unity

```diff
@@ -0,0 +1,3309 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```

Comment:
I would make a prefab for the runner that includes the cart.  Put the prefab in this folder, not in the Prafabs folder, because it is not intended to be a shared asset.